### PR TITLE
Add an extra layer of line minimization before JS and HTML minimization. 

### PR DESCRIPTION
**Pull request by mpherman2**
Hopefully a fix to the fact that line minimization on bots does not minimize as well as it does locally

### CODE REVIEW

File: src/python/bot/minimizer/html_minimizer.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
 from __future__ import absolute_import
 
 from builtins import object
+from builtins import range
```
Comment (Dor1s) [inline src/python/bot/minimizer/html_minimizer.py]: why is this needed?

Comment (mpherman2) [reply src/python/bot/minimizer/html_minimizer.py]: The linter said that I needed it.

Comment (mbarbella-chromium) [reply src/python/bot/minimizer/html_minimizer.py]: More specifically this would be needed to ensure we're using range behavior compatible with Python 3's.

**Comment by mbarbella-chromium** [comment]
Late comment: could you include comments before each of those to explain why we're doing this second pass?
